vita3k/config/src/config.cpp

```diff
@@ -132,9 +132,9 @@ static ExitCode parse(Config &cfg, const fs::path &load_path, const std::string
     get_yaml_value(config_node, "color-surface-debug", &cfg.color_surface_debug, false);
     get_yaml_value(config_node, "hardware-flip", &cfg.hardware_flip, false);
     get_yaml_value(config_node, "performance-overlay", &cfg.performance_overlay, false);
-    get_yaml_value(config_node, "backend-renderer", &cfg.backend_renderer, std::string("OpenGL"));
-    get_yaml_value(config_node, "user-id", &cfg.user_id, 0);
-    get_yaml_value(config_node, "online-id", &cfg.online_id, std::vector<std::string>{ "Vita3K" });
+    get_yaml_value(config_node, "backend-renderer", &cfg.backend_renderer, static_cast<std::string>("OpenGL"));
+    get_yaml_value(config_node, "current-user-id", &cfg.current_user_id, 0);
```

Comment:
won't this break existing configs? or does it work

Reply:
is news version so 🤔 and pr for profil is very recently, so just not important ^^ and don't clean user file, only profil created on config file, so if needed just re create it ^^

Reply:
news for this, new pr with refactor config clean all setting so change nothing ^^